packages/server/src/fhir/search.test.ts

```diff
@@ -4339,7 +4343,7 @@ describe('FHIR Search', () => {
         withTestContext(async () => {
           const parentObservations = await createObservations(repo, 3, {});
           const childObservations = [
-            await createObservations(repo, 2, { hasMember: [parentObservations[0]] }),
+            await createObservations(repo, 1, { hasMember: [parentObservations[0]] }),
```

Comment:
I'm not 100% sure, but it seems like this test depends on `childObservations[0][0]`, and when we created 2 observations, it was not always guaranteed that we would get the right one.

This test is 7 months old, and I don't think I've ever seen a failure here before today, but it failed 2 times in rapid succession today:
1. https://github.com/medplum/medplum/actions/runs/13643910874/job/38139380804?pr=6105
2. https://github.com/medplum/medplum/actions/runs/13643696777/job/38138680228?pr=6107